synapseAnnotations/data/experimentalData.json

```
@@ -977,6 +977,16 @@
         "value": "Buffy Coat",
         "description": "The middle layer of an anticoagulated blood specimen following separation by centrifugation. It contains most of the white blood cells and platelets.",
         "source": "http://purl.obolibrary.org/obo/NCIT_C84507"
+      },
+      {
+        "value":"frontal cortex",
+        "description": "Anterior portion of the neocortex, lying anterior to the central sulcus in humans. It is bounded by the parietal cortex posteriorly and the temporal cortex laterally",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0001870"
```

```suggestion
        "source": "http://purl.obolibrary.org/obo/UBERON_0001870"
```

Minor detail, but it's preferable to use the canonical IRI when possible